app/views/static_pages/privacy_policy.html.erb

```diff
@@ -62,9 +62,10 @@
     </h2>
 
     <p class="govuk-body">
-      In order for our use of your personal data to be lawful, we need to meet one (or more) conditions in the data
-      protection legislation. For the purpose of this project, your consent forms the basis of this as stated under GDPR
-      Article 6 (1)(a).
+      In order for our use of your personal data to be lawful, we need to meet one or more conditions in the data
+      protection legislation.  For the purpose of ‘<%= t("service_name") %>‘, the Department processes your
```

Comment:
Second curly quote is backwards. Not sure it matters.

Reply:
yep, should be `’`